f3/certstore.go

```diff
@@ -0,0 +1,189 @@
+package f3
+
+import (
+	"context"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/ipfs/go-datastore"
+	"github.com/ipfs/go-datastore/namespace"
+
+	"github.com/Kubuxu/go-broadcast"
+	"golang.org/x/xerrors"
+)
+
+// TODO: setup cbor-gen
```

Comment:
I think TODOs in code like this are generally unhelpful, unless they describe clearly why the thing cannot be done yet. They're generally not something that helps the reader who runs into them while spelunking.  For representing a task list, use an issue.

For early prototyping I'd be pretty relaxed, but this is going into a repo that we're working hard to pull _out_ of that prototype phase and into production code. So I think we should just hold the high standards everywhere.

Reply:
They were more of notes/TODOs for myself.